pkgs/racket-doc/scribblings/reference/data.scrbl

```diff
@@ -120,7 +120,7 @@ Returns a new immutable box that contains @racket[v].}
 Returns the content of @racket[box].}
 
 
-For any @racket[v], @racket[(unbox (box v))] returns @racket[v].
+For any @racket[v], @racket[(unbox (box v))] and @racket[(unbox (box-immutable v))] return @racket[v].
```

Comment:
Shouldn't this be `returns` (plural)?

Reply:
I shouldn't probably have commented on a merged PR. Let me move the comment to the commit.